src/lio/bin/lio_mv.c

```diff
@@ -297,6 +297,7 @@ int main(int argc, char **argv)
         while ((gop = opque_get_next_failed(q)) != NULL) {
             j = gop_get_myid(gop);
             info_printf(lio_ifd, 0, "Failed with path %s\n", flist[j].src_tuple.path);
+            status = gop_get_status(gop);
             n_errors += status.error_code;
```

Comment:
As an english speaker, `n_errors` means "number of errors"? This code doesn't seem to represent "number of errors"? What are the values supposed to represent?